00_source_data/data.py

```diff
@@ -258,5 +258,6 @@ def constant_states(states, num_constant_states, df_selection):
 overdose_copy[['County', 'State']]= overdose_deaths2.County.str.split(",", n=1, expand = True)
 overdose_copy['County']=overdose_copy['County'].str.lower()
 overdose_copy["County"]=overdose_copy["County"].str.replace('county', '')
```

Comment:
we agreed to split the county and states not to replace the states in case we have any duplicating issues

Reply:
I have created a states column prior to the white space code. There isn't a separate state column. I have splitted on the county name and created the states one. That is because there are discrepancies in the county column w.r.t their names. Some have county while others don't.

Reply:
I don't see that in the final data frame

Reply:
Did you see the last column ?